### PR TITLE
WFFHCOHORT-281 - CQL version 1.5.1 and doc updates

### PR DESCRIPTION
**Pull request by csandersdev**
I removed the SNAPSHOT from our cql_engine and
clinical_quality_framework dependencies and added some information to
the user guide on how to avoid potential PHI logging through the
configuration of the underlying log framework. While I was in the user
guide, I did a quick pass through to make sure the information was still
correct and ended up making some additional changes for correctness and
clarity.

### CODE REVIEW

**Review by deekim — APPROVED** [review]

File: docs/user-guide/getting-started.md [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
+Builds are published in [Github packages](https://github.com/orgs/Alvearie/packages?repo_name=quality-measure-and-cohort-service) or you can pull the source code and build it yourself. If you are using the precompiled artifacts, you will most likely want to start with [cohort-cli](https://github.com/Alvearie/quality-measure-and-cohort-service/packages/506888) and choose the latest shaded jar (more on that below). If you are building yourselft, use``git clone`` to pull down [the repository](https://github.com/Alvearie/quality-measure-and-cohort-service) and ``mvn install -f cohort-parent`` to build it. If you don't already have Maven installed on your workstation, [download](https://maven.apache.org/download.cgi) version 3.6.3 or newer and follow the [installation instructions](https://maven.apache.org/install.html). You should be using a Java SDK version 8.0 or higher. If you don't already have a Java SDK on your workstation, you can download one [here](https://adoptopenjdk.net/).
```
Comment (jaigoradia) [inline docs/user-guide/getting-started.md:5]: Just a nit.  Typo where you say "if you are building yourself**t**, ....

**Review by jaigoradia — APPROVED** [review]
LGTM.

File: docs/user-guide/getting-started.md (hunk 1 above)
Comment (csandersdev) [reply docs/user-guide/getting-started.md]: Good catch. I fixed it.

**Review by msargentibm — APPROVED** [review]

**Comment by msargentibm** [comment]
Screen Shot 2021-02-26 at 12 07 38 PM
Rendering of the PHI section if anyone is curious.

**Pull request merged**